includes/fs-essential-functions.php

```diff
@@ -292,15 +292,25 @@ function fs_find_caller_plugin_file() {
 	function fs_update_sdk_newest_version( $sdk_relative_path, $plugin_file = false ) {
 		global $fs_active_plugins;
 
+		$newest_sdk = $fs_active_plugins->plugins[ $sdk_relative_path ];
```

Comment:
@vovafeldman Sorry, I am not sure if I understand the issue. Could you please clarify? When the old `fs_update_sdk_newest_version` is called by a theme, `is_plugin_active` will always return `false` so `in_activation` will always be `true`. When this happens, the old SDK will be able to load/start the new SDK since the value of `$is_newest_sdk_path_valid` in `start.php` of the old SDK will be `true`.